docs/source/interactive_application.rst

```diff
@@ -83,102 +110,29 @@ Switches on and off the markers outlining the character of each anomaly
    Markers are used to indicate the left and right edges, the center,
    and the inflection point in curvature of each anomaly.
 
-Data Selection
-~~~~~~~~~~~~~~
-.. figure:: /images/parameters/data_selection_parameters.png
-   :width: 80%
-   :align: left
-
-Lines Field
-___________
-
-.. autoproperty:: peak_finder.params.PeakFinderParams.line_field
-
-Select Line
-___________
-
-.. autoproperty:: peak_finder.params.PeakFinderParams.line_id
-
-.. todo::
-
-   Add a figure showing the plan view line selection (black).
 
-:ref:`Masking Data`
-___________________
 
-.. todo::
-
-   Add a figure of a working masked result.
-
-N outward lines
-_______________
-
-Includes N lines in plan view on either side of the selected line.
-
-.. figure:: /images/parameters/data_selection/outward_line_compare.png
-   :scale: 40%
-
-   Comparing the plan view with 1 outward line (left) and 2 outward lines
-   (right).
-
-Flip Y (-1x)
-____________
-
-.. autoproperty:: peak_finder.params.PeakFinderParams.flip_sign
-
-.. todo::
-
-   Update docstring and add figure showing the effect of flipping y.
-
-Select group colors
-___________________
-
-.. todo::
-
-   Add figure of color picker widget.  Move this ui to visualization group?
-
-Detection Parameters
-~~~~~~~~~~~~~~~~~~~~
+Output Parameters
+~~~~~~~~~~~~~~~~~
 
-The detection parameters are those that the peak-finder application uses to
-tune the characterization and detection of anomalies within the data.  Most
-of these are already described in the :ref:`Methodology` section.  Follow
-the links for detailed descriptions of each parameter.
+Create trend line
+__________________
 
-.. figure:: /images/parameters/detection_parameters.png
-   :align: left
+Run a trend line detection algorithm on the result of the Peak Finder algorithm. Results are stored as a curve object in the geoh5 file
+with the same group ID as the Peak Finder result.
 
 
-- :ref:`Smoothing window <Smoothing>`
-- :ref:`Minimum Amplitude (%) <Minimum Amplitude>`
-- :ref:`Minimum Value <Minimum Data Value>`
-- :ref:`Minimum Width (m) <Minimum Width>`
-- :ref:`Max Peak Migration (m) <Maximum Peak Migration>`
-- :ref:`Minimum # Channels <Minimum number of channels>`
-- :ref:`Merge # Peaks <Merge N Peaks>`
-- :ref:`Max Separation <Max Group Separation>`
+ figure:: /images/parameters/visualization/trend_lines.png
```

Comment:
link not working:
![image](https://github.com/user-attachments/assets/1044ce21-dad6-49c8-95c8-61b045d61d91)